src/Components/Card/index.jsx

```diff
@@ -2,23 +2,29 @@ import axios from "axios";
 import React from "react";
 import "./styles.css";
 import { useNavigate } from "react-router-dom";
+
 import { useAuth } from "../../Contexts/AuthContext";
 
 import { useCart } from "../../Contexts/CartContext";
```

Comment:
Suggestion - All these Api calls files can be imported into a index.js and imported together. Code look's cleaner and easy to read.

Reply:
Thanks for the suggestion i will do that in the next pr